src/Imath/ImathEuler.h

```diff
@@ -149,9 +157,10 @@ template <class T> class Euler : public Vec3<T>
         YZYr = 0x1010,
         ZYZr = 0x0110,
         ZXZr = 0x0010,
-        //          ||||
-        //          VVVV
-        //  Legend: ABCD
+        //       ||||
+        //       VVVV
+        ///      ABCD
+        // Legend: 
```

Comment:
this is not going to be rendered in the docs in a way that makes any sense

Reply:
Yeah, the hex values don't show up, so I intended to leave these for just the header. The /// before the ABCD was a typo.